app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -73,6 +74,16 @@ def review_submitted
       @application_form = current_application
     end
 
+    def review_previous_application
+      if @application_form.nil?
```

Comment:
This can never be true, because if the application form doesn't exist it'll raise a `ActiveRecord::RecordNotFound` exception.